src/main/java/org/jabref/gui/preferences/file/FileTabViewModel.java

```diff
@@ -84,6 +85,10 @@ public void setValues() {
         secondarySortFieldsProperty.addAll(fieldNames);
         tertiarySortFieldsProperty.addAll(fieldNames);
 
+        primarySortFieldsProperty.sort(Comparator.comparing(Field::getDisplayName));
```

Comment:
Simple solution but not effective. You are essentially doing the same sort 3 times.
I would create: `List<Field> fieldNames = new ArrayList<>(FieldFactory.getCommonFields())`
Then sort it and add it to the properties